src/main/java/com/j256/ormlite/misc/functional/Supplier.java

```diff
@@ -0,0 +1,14 @@
+package com.j256.ormlite.misc.functional;
+
+/**
+ * This is a functional interface intended to support lambda expressions in ormlite.
+ */
+public interface Supplier<T> {
+
+    /**
+     * Gets a value.
```

Comment:
Please remove the `@return` and explain what the method does.  Should it take the ID as an argument?